src/aviary/tools/base.py

```diff
@@ -344,6 +344,15 @@ class Tool(BaseModel):
             " serialization, and the validation alias enables deserialization."
         ),
     )
+    concurrency_safe: bool = Field(
+        default=True,
+        exclude=True,
```

Comment:
Isn't this `exclude` sort of a problem, as we serialize/deserialize tools? When we do so, we will lose our `concurrency_safe`-ness